web/src/components/AssertionForm/AssertionFormConfirmModal.tsx

```diff
@@ -0,0 +1,22 @@
+import {Modal} from 'antd';
```

Comment:
We are now alerting the user that there is a version mismatch before opening the assertion form